File: 8-Boolean-and-Conditional-Logic/12-Calling-In-Sick.py
from random import randint, choice

actually_sick = choice([True, False])
kinda_sick = choice([True, False])
hate_your_job = choice([True, False])
sick_days = randint(0, 10)

calling_in_sick = None

if actually_sick and sick_days > 0:
    calling_in_sick = True
elif kinda_sick and hate_your_job and sick_days > 0:
    calling_in_sick = True
else:
    calling_in_sick = False

File: 8-Boolean-and-Conditional-Logic/12-Calling-In-Sick.js
function choice(arr) {
  return arr[Math.floor(Math.random() * arr.length)]
}

function randint(min, max) {
  return Math.floor(Math.random() * (max - min + 1)) + min
}

let actuallySick = choice([true, false])
let kindaSick = choice([true, false])
let hateYourJob = choice([true, false])
let sickDays = randint(0, 10)

let callingInSick = null

if (actuallySick && sickDays > 0) {
  callingInSick = true
}
else if (kindaSick && hateYourJob && sickDays > 0) {
  callingInSick = true
}
else {
  callingInSick = false
}